dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/TrackedEntitiesExportControllerTest.java

```diff
@@ -1014,18 +1014,14 @@ void getAttributeValuesFileByAttributeAndProgramIfFileIsNotFound() {
     enroll(trackedEntity, program, orgUnit);
 
     String fileUid = CodeGenerator.generateUid();
-    TrackedEntityAttribute tea =
-        addProgramAttributeValue(trackedEntity, program, ValueType.FILE_RESOURCE, fileUid);
 
-    assertStartsWith(
-        "FileResource with id " + fileUid,
-        GET(
-                "/tracker/trackedEntities/{trackedEntityUid}/attributes/{attributeUid}/file?program={programUid}",
-                trackedEntity.getUid(),
-                tea.getUid(),
-                program.getUid())
-            .error(HttpStatus.NOT_FOUND)
-            .getMessage());
+    IllegalQueryException illegalQueryException =
```

Comment:
```suggestion
    IllegalQueryException exception =
```

Reply:
Why do we now get a `IllegalQueryException` if the file is not found? I think this should be a 404 🤔

Reply:
Yes, it should.
But the whole `trackedEntityAttributeValueService` is using unchecked exceptions.
I can fix it in a next PR.